### PR TITLE
Create an movie brunch

### PR DESCRIPTION
**Pull request by GuhaPritam**
please chaked this

### CODE REVIEW

File: muvie/data/users.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "users": [
```
Comment (atapas) [inline muvie/data/users.json]: Why users? It is a list of movies right?

File: muvie/data/users.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            "rating": "5 star",
+            "director": "rox"
+        },
+        {
+            "movie photo": "02",
+            "name": "joker 2020",
+            "rating": "5 star",
+            "director": "rox"
+        },
+        {
```
Comment (atapas) [inline muvie/data/users.json]: Put proper photo url

File: muvie/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>muvie</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div id="pritam-id">
```
Comment (atapas) [inline muvie/index.html]: What is pritam-id? Change something to meaningful.

File: muvie/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+
+function fetchUserData() {
+
+
+    fetch('./data/users.json')
```
Comment (atapas) [inline muvie/index.js]: Why the file name is users.json?

File: muvie/index.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        .then(response => response.json())
+            .then(data => {
+                // All users
+                const users = data.users;
```
Comment (atapas) [inline muvie/index.js]: Why users is the variable name?

**Review by atapas — CHANGES_REQUESTED** [review]
@Guha-Pritam, please make changes by looking into my comments. Also paste a screenshot of the app in the comment. Please send for review once done.

File: muvie/data/users.json (hunk 1 above)
Comment (GuhaPritam) [reply muvie/data/users.json]: yes

File: muvie/index.js (hunk 5 above)
Comment (GuhaPritam) [reply muvie/index.js]: what name I put there?

File: muvie/index.js (hunk 4 above)
Comment (GuhaPritam) [reply muvie/index.js]: can i change the name to index.json?

File: muvie/index.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+
+function fetchUserData() {
+
+
+    fetch('./data/movies.json')
+        .then(response => response.json())
+            .then(data => {
+                // All users
+                const users = data.users;
```
Comment (atapas) [inline muvie/index.js]: Why data.users? The JSON it is movies array?

File: muvie/index.js (hunk 5 above)
Comment (atapas) [reply muvie/index.js]: Is it related to users?

**Review by atapas — CHANGES_REQUESTED** [review]

**Comment by GuhaPritam** [comment]
now i push the orijin and please checked this

**Review by GuhaPritam — COMMENTED** [review]
pleace checked this

File: muvie/index.html [hunk 7]
```diff
@@ -1,20 +1,17 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>work 2</title>
+    <title>muvie</title>
```
Comment (atapas) [inline muvie/index.html:7]: Movie spelling is still wrong.

**Review by atapas — CHANGES_REQUESTED** [review]

**Comment by atapas** [comment]
@Guha-Pritam  why  have you added a duplicate label?

**Pull request merged**